### ISSUE TITLE
Error message on compiling phar 

### ISSUE THREAD

**Issue by oburk** [p1ho/accessibility-checker#5]
Hi,

I got the following error message when I try to build a phar file from my project:

```
? Dumping the Composer autoloader
 In ComposerOrchestrator.php line 76:
                                   
   Could not dump the autoloader.  
                                   
 In ComposerOrchestrator.php line 76:
                                                                                
   The command "'/bin/composer' 'dump-autoload' '--classmap-authoritative' '--  
   no-dev'" failed.                                                             
                                                                                
   Exit Code: 1(General error)                                                  
                                                                                
   Working directory: /tmp/box/Box29462                                         
                                                                                
   Output:                                                                      
   ================                                                             
   Generating optimized autoload files (authoritative)                          
                                                                                
                                                                                
   Error Output:                                                                
   ================                                                             
                                                                                
                                                                                
                                                                                
                                                                                
     [RuntimeException]                                                         
                                                                                
                                                                                
     Could not scan for classes inside "/tmp/box/Box29462/vendor/p1ho/accessib  
   ility-checker/tests/ColorContrast/CheckerTest.php" which does not appear to  
    be a file nor a folder   
```

# Steps for testing:

run composer
```
composer install
```
get phar compiler
```
curl -L --output box.phar https://github.com/humbug/box/releases/download/3.8.4/box.phar
```
run compiler
```
php box.phar compile
```

I created a pull request #4  which solves the problems on my side.

**Comment by p1ho** [comment]
Hello, thanks for opening the issue, I apologize I haven't been able to get back to you sooner!
Can you please let me know your OS and your php version?
I'll see if I can reproduce the issue!

**Comment by oburk** [comment]
Hello, I am developing on a MS Windows 10 system with PHP 7.4

**Comment by p1ho** [comment]
Awesome, I'll try it out when I'm free, thanks!

**Comment by p1ho** [comment]
Hello, I was able to reproduce the issue, could you please give the steps you took to resolve it? I looked at the PR you opened and tried the solution on my local repository (using [this technique](https://stackoverflow.com/questions/29994088/composer-require-local-package)), ~but was unable to get rid of the error.~ (Edit: Nevermind, stupid mistake, it works)

**Comment by p1ho** [comment]
Merged and Closed #4 Thank you for finding this bug!
